mvc/src/main/java/nextstep/mvc/controller/tobe/HandlerExecution.java

```diff
@@ -2,11 +2,20 @@
 
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
+import java.lang.reflect.Method;
+import nextstep.mvc.view.JspView;
 import nextstep.mvc.view.ModelAndView;
 
 public class HandlerExecution {
 
+    private final Method method;
+
+    public HandlerExecution(Method method) {
+        this.method = method;
+    }
+
     public ModelAndView handle(final HttpServletRequest request, final HttpServletResponse response) throws Exception {
-        return null;
+        final Object controller = method.getDeclaringClass().getConstructors()[0].newInstance();
```

Comment:
```suggestion
        final Object controller = method.getDeclaringClass().getDeclaredConstructor().newInstance();

```
해당 생성자를 직접 호출하는 방법도 있습니다!